lib/responders.rb

```diff
@@ -1,11 +1,5 @@
-require 'action_controller'
```

Comment:
All `config.action_controller` configurations are copied to `ActionController::Base.config` when ActionController is loaded: https://github.com/rails/rails/blob/4-2-stable/actionpack/lib/action_controller/railtie.rb#L54-L60
If responders loads ActionController too early, we cannot set ActionController configurations **in the initializers phase** because the copying is already performed.

Why do you need to `require "action_controller"` in the such early phase (i.e., before the initializers phase) ?
If you'd like to extend ActionController, you should apply the extension when ActionController is loaded.
`ActiveSupport.on_load(:action_controller)` will help it, you know.